src/Tools/IdeCoreBenchmarks/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using AnalyzerRunner;
+using BenchmarkDotNet.Attributes;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Diagnosers;
+using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Validators;
+using Microsoft.CodeAnalysis.MSBuild;
+
+namespace IdeCoreBenchmarks
+{
+    internal class Program
+    {
+        private class IgnoreReleaseOnly : ManualConfig
```

Comment:
Is this used anywhere?

Reply:
Not in the code, but I used it for debugging. Will be useful when writing new benchmarks